repositories/src/test/scala/cromwell/pipeline/datastorage/dao/mongo/DocumentRepositoryTest.scala

```diff
@@ -1,17 +1,61 @@
 package cromwell.pipeline.datastorage.dao.mongo
 
+import akka.http.scaladsl.testkit.ScalatestRouteTest
+import com.dimafeng.testcontainers.MongoDBContainer
+import com.dimafeng.testcontainers.scalatest.TestContainerForAll
+import cromwell.pipeline.database.TestMongoEngine
+import cromwell.pipeline.datastorage.dao.repository.ProjectConfigurationRepository
+import cromwell.pipeline.datastorage.dao.utils.TestProjectUtils
+import cromwell.pipeline.datastorage.dto._
+import cromwell.pipeline.utils.{ MongoConfig, TestContainersUtils }
 import org.mockito.Mockito.when
 import org.mongodb.scala.result.UpdateResult
 import org.mongodb.scala.{ Document, MongoCollection }
-import org.scalatest.{ AsyncWordSpec, Matchers }
+import org.scalatest.{ Matchers, WordSpec }
 import org.scalatestplus.mockito.MockitoSugar
 
-class DocumentRepositoryTest extends AsyncWordSpec with Matchers with MockitoSugar {
+import java.nio.file.Paths
+import scala.concurrent.Await
+import scala.concurrent.duration.Duration
 
-  private val updateResult = mock[UpdateResult]
-  private val collection = mock[MongoCollection[Document]]
+class DocumentRepositoryTest
+    extends WordSpec
```

Comment:
Why did you change `AsyncWordSpec` to `WordSpec`? All tests here are using `Future[Assertion]`